lib/admin/WP_Auth0_Admin_Advanced.php

```diff
@@ -461,8 +461,9 @@ public function migration_ws_validation( array $old_options, array $input ) {
 		}
 
 		try {
-			$token_decoded               = JWT::decode( $input['migration_token'], $secret, [ 'HS256' ] );
-			$input['migration_token_id'] = isset( $token_decoded->jti ) ? $token_decoded->jti : null;
+			$signature_verifier          = new WP_Auth0_SymmetricVerifier( $secret );
```

Comment:
It looks similar to the file `WP_Auth0_Routes`. Should it be extracted?

Reply:
This logic will likely be removed as part of the major, didn't want to go too far in this PR.